src/invidious/yt_backend/connection_pool.cr

```diff
@@ -45,24 +41,21 @@ struct YoutubeConnectionPool
 
   private def build_pool
     DB::Pool(HTTP::Client).new(initial_pool_size: 0, max_pool_size: capacity, max_idle_pool_size: capacity, checkout_timeout: timeout) do
-      conn = HTTP::Client.new(url)
-      conn.family = CONFIG.force_resolve
-      conn.family = Socket::Family::INET if conn.family == Socket::Family::UNSPEC
-      conn.before_request { |r| add_yt_headers(r) } if url.host == "www.youtube.com"
-      conn
+      next make_client(url, force_resolve: true)
```

Comment:
This line fails Ameba's `Style/RedundantNext` rule.

Should it also be disabled like `Style/RedundantBegin` and `Style/RedundantReturn`?

Reply:
I didn't know that next could be used like that! And yes, this will make the syntax consistent with the rest.